notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb

```diff
@@ -0,0 +1,1586 @@
+{
```

Comment:
<p>[Not blocking] There is one more feature(<code>keyField</code>) in  <a href="https://source.corp.google.com/piper///depot/google3/google/cloud/aiplatform/master/batch_prediction_job.proto;bpv=1;bpt=1;l=173?q=f:cloud%2Fai%20f:master%20f:batch&amp;ct=os&amp;sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental&amp;gsn=InstanceConfig&amp;gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fgoogle3%3Flang%3Dprotobuf%3Fpath%3Dgoogle%2Fcloud%2Faiplatform%2Fmaster%2Fbatch_prediction_job.proto%234.0.3.1" target="_blank">InstanceConfig</a> message that would simplify the output format and make it easier for customer to further join the prediction result by id.</p><p> </p><p>And if we could have </p><p> </p><p><code></code>`</p><p>KEY_FIELDS = ID_COLUMN_NAME</p><p> </p><p>import json</p><p> </p><p>request_with_excluded_fields = {</p><p>&nbsp;...</p><p> </p><p>&nbsp;&nbsp;"instanceConfig": {"excludedFields": EXCLUDED_FIELDS,"keyField":"KEY_FIELDS" },</p><p>}</p><p> </p><p>with open("request_with_excluded_fields.json", "w") as outfile:</p><pre class="ql-syntax" spellcheck="false">&nbsp;&nbsp;json.dump(request_with_excluded_fields, outfile)
<code></code>`
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>